kolibri/core/device/test/test_api.py

```diff
@@ -204,6 +206,16 @@ def test_urls(self):
             # Make sure each url is a valid link
             self.assertTrue(url.startswith('http://'))
 
+    @patch('kolibri.core.device.api.get_urls', return_value=(1, ['http://127.0.0.1:8000', 'http://kolibri.com']))
+    def test_no_localhost_urls_when_others_available(self, get_urls_mock):
+        response = self.client.get(reverse('deviceinfo'), format="json")
+        self.assertEqual(len(response.data['urls']), 1)
```

Comment:
how do we know that `http://127.0.0.1:8000` was not returned?

Reply:
True.